warehouse/admin/templates/admin/banners/list.html

```diff
@@ -22,7 +22,7 @@
 {% block content %}
 
 <div class="card card-primary card-outline">
-  {% if request.has_permission('psf_staff') %}
+  {% if request.has_permission(Permissions.AdminBannerWrite) %}
```

Comment:
This is probably the simplest example where we currently have a principal used in the permission string. Instead of using `"psf_staff"` as a permission, use a more accurate string `"admin:banner:write"`.

Thus we convey what is needed, instead of who we expect to have it - decoupling the conceptual group from the permission.